modules/github-bots/main.tf

```diff
@@ -21,12 +21,55 @@ variable "broker" {
   type        = map(string)
 }
 
-variable "source_code" {
-  description = "The source code for the bot."
-  type = object({
-    working_dir = string
-    importpath  = string
-  })
+variable "containers" {
```

Comment:
I would love to be able to share this structure somehow... 😭

Reply:
yeah - I'll take a look next week and see if we can make this a bit more DRY